repair/row_level.cc

```diff
@@ -1417,9 +1413,16 @@ class repair_meta {
             return make_ready_future<>();
         }
         stats().rpc_call_nr++;
+        // Even though remote partitioner name is ignored in the current version of
+        // repair, we still have to send something to keep compatibility with nodes
+        // that run older versions. This will make it possible to run mixed cluster.
+        // Murmur3 is appropriate because that's the only supported partitioner at
+        // the time this change is introduced.
```

Comment:
We were lucky with this, not to have to send the partitioner name everywhere just for compatibility.

Reply:
Lucky because we removed all other partitioners?

Reply:
Lucky that it turned out we don't need to do complicated acrobatics for something that nobody uses.